src/component/header/header/Search.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from "react";
+import { getArticles } from "../../../api";
+
+const Search = ({ allArticles, setAllArticles }) => {
+  const [inputSearchValue, setInputSearchValue] = useState("");
+  const pressedToSearch = () => {
```

Comment:
This is a lot more that what the ticket requires? I'd advise keeping search functionality as a stretch goal.